site/hi/index.html

```diff
@@ -0,0 +1,669 @@
+﻿<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2018 Goldman Sachs and others
```

Comment:
Fix the copyright. Please follow the instructions here: https://www.eclipse.org/projects/handbook/#ip-copyright-headers
Refer to the comments about "owner"

Reply:
fixed

Reply:
just changed year to 2020 as not understood link.
Is whole copyright should be in new format as below ?

/********************************************************************************
 * Copyright (c) {year} {owner}[ and others] 1
 *
 * This program and the accompanying materials are made available under the 2
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0 3
 *
 * Contributors: 4
 *   {name} - initial API and implementation
 ********************************************************************************/